extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -100,9 +94,9 @@ export class Bind {
     };
 
     this.ampdoc.whenBodyAvailable().then(body => {
-      const {bindings, expressions} = this.scanForBindings_(body);
-      this.bindings_ = bindings;
-      this.evaluator_ = new BindEvaluator(expressions);
+      const {boundElements, evaluatees} = this.scanForBindings_(body);
```

Comment:
Do you think `whenFirstVisible` is more appropriate? Agree that we should make sure it doesn't affect time-to-first-paint.